scripts/wsrep_sst_common.sh

```diff
@@ -293,6 +310,14 @@ wsrep_log_debug()
     fi
 }
 
+#
+# Exit with the provided exit code or global override if defined.
+#
+safe_exit() {
```

Comment:
isn't it better to just return EAGAIN at until the datadir deletion/modification point? Global override can have other side affects if it is set again somewhere in middle.

if we know for sure, we will return EAGAIN until datadir, then it is better to remove other error codes until datadir deletion code/SST start part.
currently we return codes 1,2,22,32 etc. so after your patch, we can return only 22 until SST start.

Reply:
Another question, is the driver SST code taking actions based on error codes 1&2,32? now that  we override with 22, we may have to adjust if there is code written to handle 1,2 etc.